src/main/java/com/appcharge/server/service/impl/OfferServiceImpl.java

```diff
@@ -57,6 +71,8 @@ public ResponseEntity<?> createOffer() throws IOException {
                     JsonNode.class
             );
             JsonNode responseBody = response.getBody();
+            updateOfferId();
+            assignNewOrderToPlayers(responseBody);
```

Comment:
Why we are using this? I think we deleted it on the huuuge mocker.

Reply:
fixed